src/Header/Nav/Nav.module.scss

```diff
@@ -133,52 +220,44 @@ $colour-nav-text: $colour-text-inverse;
       }
     }
   }
-}
-
-.myAccount {
-  border-top: 1px solid $colour-nice-white;
-
-  @include mq($from: md) {
-    display: none;
-  }
-}
-
-.myAccountHeading {
-  border: rem($spacing-x-small) solid $colour-nav-background;
-  font-size: 100%;
-  margin: 0;
-  padding: rem($spacing-small $spacing-medium);
-}
-
-.menu {
-  list-style: none;
-  margin: auto;
-  padding: 0;
 
-  li {
-    margin: 0;
+  .subMenuWrapper {
+    border-bottom: rem($spacing-xx-small) solid $colour-nav-background;
+    bottom: 0;
+    height: rem($sub-nav-height);
+    left: 0;
+    position: absolute;
+    width: 100%;
   }
 
-  @include mq($from: md) {
-    // Fallback for non flexbox browsers that support media queries e.g. IE9
-    display: table;
+  .subMenuList {
+    height: 100%;
+    margin: auto;
     max-width: rem($nice-container-max-width);
     // TODO Replace with mixin from nice design system
     width: 96%;
     width: calc(100% - #{rem($spacing-large)});
 
     li {
-      display: table-cell;
+      display: inline-block;
+      height: 100%;
     }
 
-    @supports (display: flex) {
-      display: flex;
-      flex-direction: row;
-      flex-wrap: nowrap;
-      justify-content: space-between;
+    /* stylelint-disable */
+    a {
+      /* stylelint-enable */
+      display: block;
+      height: 100%;
+      padding-bottom: 0;
+      padding-top: 0;
 
-      li {
-        flex-grow: 1;
+      &[aria-current='true'],
+      &[aria-current='page'] {
+        background: transparent;
+
```

Comment:
Do we need a new line here?

Reply:
Yes, because the next block is a `&:after`